gcda11y-survey.html

```diff
@@ -385,341 +391,104 @@ <h2>Technical notes</h2>
-			<div class="col-md-4 col-xs-12">
-				<p id="q6-10">Resources</p>
-			</div>
-			<div class="col-md-2 col-sm-3 col-xs-12">
-				<input id="q6-10a" name="q6-10" type="radio" value="Not important" aria-labelledby="q6-ni" aria-describedby="q6-10 q6-desc" />
-				<label for="q6-10a" class="hidden-md hidden-lg">Not important</label>
-			</div>
-			<div class="col-md-2 col-sm-3 col-xs-12">
-				<input id="q6-10b" name="q6-10" type="radio" value="Important" aria-labelledby="q6-i" aria-describedby="q6-10 q6-desc" />
-				<label for="q6-10b" class="hidden-md hidden-lg">Important</label>
-			</div>
-			<div class="col-md-2 col-sm-3 col-xs-12">
-				<input id="q6-10c" name="q6-10" type="radio" value="Very important" aria-labelledby="q6-vi" aria-describedby="q6-10 q6-desc" />
-				<label for="q6-10c" class="hidden-md hidden-lg">Very important</label>
-			</div>
-			<div class="col-md-2 col-sm-3 col-xs-12">
-				<input id="q6-10d" name="q6-10" type="radio" value="Not applicable" aria-labelledby="q6-na" aria-describedby="q6-10 q6-desc" />
-				<label for="q6-10d" class="hidden-md hidden-lg">Not applicable</label>
-			</div>
-		</div>
-
-<!-- Question 6.11: -->
-		<div class="row" role="group" aria-labelledby="q6-11">
-			<div class="col-md-4 col-xs-12">
-				<p id="q6-11">Procurement</p>
-			</div>
-			<div class="col-md-2 col-sm-3 col-xs-12">
-				<input id="q6-11a" name="q6-11" type="radio" value="Not important" aria-labelledby="q6-ni" aria-describedby="q6-11 q6-desc" />
-				<label for="q6-11a" class="hidden-md hidden-lg">Not important</label>
-			</div>
-			<div class="col-md-2 col-sm-3 col-xs-12">
-				<input id="q6-11b" name="q6-11" type="radio" value="Important" aria-labelledby="q6-i" aria-describedby="q6-11 q6-desc" />
-				<label for="q6-11b" class="hidden-md hidden-lg">Important</label>
-			</div>
-			<div class="col-md-2 col-sm-3 col-xs-12">
-				<input id="q6-11c" name="q6-11" type="radio" value="Very important" aria-labelledby="q6-vi" aria-describedby="q6-11 q6-desc" />
-				<label for="q6-11c" class="hidden-md hidden-lg">Very important</label>
-			</div>
-			<div class="col-md-2 col-sm-3 col-xs-12">
-				<input id="q6-11d" name="q6-11" type="radio" value="Not applicable" aria-labelledby="q6-na" aria-describedby="q6-11 q6-desc" />
-				<label for="q6-11d" class="hidden-md hidden-lg">Not applicable</label>
-			</div>
-		</div>
-	
-	
-	</fieldset>
-
-	
-	
-	
-	
-	
+			
+			<table class="table">
+				<caption class="wb-inv">
+				***Titre
+				</caption>
+				<thead>
+				  <tr>
+					<th>Catégorie**</th>
+					<th id="q6-ni">Not important</th>
+					<th id="q6-i">Important</th>
+					<th id="q6-vi">Very important</th>
+					<th id="q6-na">Not applicable</th>
+				  </tr>
+				</thead>
+				<tbody>
+				  <tr>
+					<td id="q6-01">Produit de communication</td>
+					<td><label for="q6-01a"><input id="q6-01a" name="q6-01" type="radio" value="Not important" aria-labelledby="q6-ni" aria-describedby="q6-01 q6-desc" /></label></td>
```

Comment:
Parce que tu utilise "aria-labellledby" il est innutile d'avoir l'élément `label` car il n'offre aucun avantage car nous utilisons une méthode alternative pour rencontrer l'accessibilité.

Peut-être il serait utile d'inclure un texte dans le label qui double l'entête du tableau. On pourrait le pâlir en utilisant les classe d'opactité. https://wet-boew.github.io/wet-boew-styleguide/design/opacity-en.html et si nécessaire l'inclure dans un span avec le role "presentation" car c'est sa fonction et on pourrait inclure l'id du label dans liste des labelledby afin d'améliorer la conformité à "Label in name" -> https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html